### ISSUE TITLE
chore(jans-client-api): remove jans-client-api project

### ISSUE THREAD

**Issue by yuriyz** [JanssenProject/jans#2582]
-------------------

### Description

Remove jans-client-api project from jans repository.

Due to lack of interest in the project and continous effort to support it, it was decided to remove it.

-------------------

### Prepare

- [ ] Read contribution guidelines
- [ ] Read license information

------------------

### Test cases and code coverage

- [ ] Write unit test to cover added/changed code
- [ ] Update integration tests to cover added/changed code

-------------------

### Document the changes

- [ ] task for updating user guides if needed
- [ ] task for updating installation and configuration guides if needed 
- [ ] task for updating developer documentation if needed
- [ ] task for updating technical documentation if needed

**Comment by ossdhaval** [comment]
@yuriyz we will need a task for `cleaning up labels`

**Comment by yuriyz** [comment]
@ossdhaval feel free to create additional ticket if it's really needed

**Comment by moabu** [comment]
> @yuriyz we will need a task for `cleaning up labels`

@ossdhaval this will be done in the ci/cd step/ Since we have a json removing it will cause the label to be removed. I got that one

**Comment by iromli** [comment]
@moabu Regarding this task:

> Remove client api docker setup resources @iromli

The configs,secrets, and data insertion is executed when running `janssen/client-api` container. Hence in fresh CN setup, if there's no `client-api` pod, there will be no references to client-api in CN, except this scope:

```
dn: inum=6D90,ou=scopes,o=jans
description: jans_client_api scope which is required to call jans_client_api API
displayName: jans_client_api
```

which is taken from `jans-linux-setup/jans_setup/templates/scopes.ldif` (synchronized to `janssen/persistence-loader`).

When the upstream template has been changed, we can sync the template into `janssen/persistence-loader` (the image needs the commit ID that contains the changes, usually from `main` branch) and rebuild the persistence image.

**Comment by moabu** [comment]
> @moabu Regarding this task:
> 
> 
> 
> > Remove client api docker setup resources @iromli
> 
> 
> 
> The configs, secrets, and data insertion is executed when running `janssen/client-api` container. Hence in fresh CN setup, if there's no `client-api` pod, there will be no references to client-api in CN, except this scope:
> 
> 
> 
> ```
> 
> dn: inum=6D90,ou=scopes,o=jans
> 
> description: jans_client_api scope which is required to call jans_client_api API
> 
> displayName: jans_client_api
> 
> ```
> 
> 
> 
> which is taken from `jans-linux-setup/jans_setup/templates/scopes.ldif` (synchronized to `janssen/persistence-loader`).
> 
> 
> 
> When the upstream template has been changed, we can sync the template into `janssen/persistence-loader` (the image needs the commit ID that contains the changes, usually from `main` branch) and rebuild the persistence image.

Yes, sounds good . The reference commit id can be updated after we merge the associated pr as there is no code changes

**Comment by iromli** [comment]
@moabu The commit 403ef1d5702158764af8a82aeeb6eff3269fe38d removed the scope for `jans_client_api`.

I rebuild the persistence-loader image locally to use templates from commit above and worked as expected in CN.
The question is will the commit ID 403ef1d5702158764af8a82aeeb6eff3269fe38d available after PR 2597 merged to `main` branch (considering we will do squash-and-merge)?

**Comment by iromli** [comment]
If the commit ID is not available after merged to `main` branch, we need to use the one from `main`.

> The question is will the commit ID https://github.com/JanssenProject/jans/commit/403ef1d5702158764af8a82aeeb6eff3269fe38d available after PR 2597 merged to main branch (considering we will do squash-and-merge)?
